bolt/test/AArch64/incorrect-negate-ra-state.s

```diff
@@ -0,0 +1,41 @@
+# RUN: llvm-mc -filetype=obj -triple aarch64-unknown-unknown %s -o %t.o
```

Comment:
Can you add here a brief description of the test, explaining why this case is wrong.